src/styles/blocks/page.scss

```diff
@@ -0,0 +1,70 @@
+.page {
+  font-family: Poppins, sans-serif;
+  background-color: $dark-grey;
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+
+    &--with-menu {
+      overflow: hidden;
+    }
+  }
+
+  &__menu {
+    position: fixed;
```

Comment:
I can see pictures above the menu
<img width="1511" alt="image" src="https://user-images.githubusercontent.com/99652389/180950795-63b489dc-8007-4a40-9ae3-0afde66ff8ec.png">